packages/dev/loaders/src/glTF/2.0/Extensions/MSFT_lod.ts

```diff
@@ -18,7 +18,7 @@ declare module "../../glTFFileLoader" {
         /**
          * Defines options for the MSFT_lod extension.
          */
-        [NAME]?: Partial<{
+        ["MSFT_lod"]?: Partial<{

```

Comment:
So we can't use the `NAME` variable (runtime restriction), but can we reference the type? E.g. could we still guarantee this string doesn't get out of sync with the `NAME` variable via:
```suggestion
        ["MSFT_lod" satisfies typeof NAME]?: Partial<{
```

Reply:
Talked to @RaananW, this doesn't work for the same reason. It's actually the `NAME` type that is inaccessible, not the value.